src/components/organisms/EventModal/EventModal.scss

```diff
@@ -24,13 +24,6 @@ $EventModal--padding-top: 40px;
     opacity: 0.8;
   }
 
-  &__description {
```

Comment:
Why was this removed?

Reply:
because was not used in that version. now brought back (although I do not see any effect of it when used as `div`)